cli_reference/moactl/moactl.adoc

```diff
@@ -0,0 +1,8 @@
+[id="moactl-reference"]
+= moactl reference
+include::modules/common-attributes.adoc[]
+:context: moactl-reference
+
+This is the moactl docs
```

Comment:
@lamek - should this have stayed in?

Reply:
I followed up on this here - https://github.com/openshift/openshift-docs/pull/26105